processing.py

```diff
@@ -1,11 +1,33 @@
 from prefect import flow, task, get_run_logger
 from tiled.client import from_profile
+from tpx3awkward import extract_fpaths_from_sid, raw_to_sorted_df
 
 tiled_client = from_profile("nsls2")["chx"]
 tiled_client_chx = tiled_client["raw"]
 tiled_cilent_sandbox = tiled_client["sandbox"]
 tiled_cilent_processed = tiled_client["processed"]
 
+@task
+def get_df_uncent(run):
+    sid = run['start']['scan_id']
+    raw_file_paths = extract_fpaths_from_sid(sid)
+    for file in raw_file_paths:
+        if (os.path.exists(file)):
+            yield raw_to_sorted_df(file)
+
+@flow
```

Comment:
This also can be a normal python function, since this is called within another task.
```suggestion
```